app/models/site_history.rb

```diff
@@ -1,7 +1,35 @@
 # frozen_string_literal: true
 
 class SiteHistory < ApplicationRecord
+  include DateValidateable
+
   belongs_to :planning_application
 
-  validates :date, :application_number, :description, :decision, presence: true
+  alias_attribute :reference, :application_number
```

Comment:
I aliased this because elsewhere in the app we use reference but have the label as "Application number" - not sure what the design guidance is for this?